about.html

```diff
@@ -0,0 +1,158 @@
+        <li><a href="index.html" class="active">Home</a></li>
+        <li><a href="">Program</a></li>
+        <li><a href="">Join</a></li>
+        <li><a href="">Sponsor</a></li>
+        <li><a href="">News</a></li>
+        <li><button class="btn-1">Tech Campaign</button></li>
+      </ul>
+    </div>
+
+    <div id="navbar">
+      <ul class="nav-menu">
+          <li class="nav-item nav-area"><a href="index.html" class="nav-link">Home</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+      </ul>
+
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </div>
+  
+      <div class="bg-content">
+        <div class="main-section">
+          <div class="heading">
+            <p class="header-paragraph">"Computer Workshop"</p>
+            <h1>MAINTENANCE AND SOFTWARE INSTALLATION</h1>
+          </div>
+      
+          <div class="header-text">
+            <p class="white-bg">We provide basic computer maintenance services and we also install softwares of all kinds
+              both system and application softwares
+            </p>
+          </div>
+          
+          <div>
+            <p class="center-text">Please contact us through email for further
+              questions about our services and opearations using this email
+              <strong>shaaibusuleiman9@gmail.com</strong>
+            </p>
+          </div>
+        </div>
+      </div>
+      
+    </div>
+
+    <section>
+      <div>
+        <p class="global center-text">Global Tech Solutions And Innovations</p>
+        <div class="program-line"></div>
+        <p class="center-text">We provide high quality services at affordable rates with long lasting guarantee.</p>
+      </div>
+
+      <div class="global-image">
+        <div class="global-container">
+          <span class="material-symbols-outlined">
+            travel_explore
+            </span>
+            <p class="global-text">Global Tech Solutions and Innovations</p>
+        </div>
+      </div>
+
+      <hr>
+
+      <div class="global-summit">
+        <p class="global-summit-text">Milestone We Have Made as a company</p>
+        <div class="program-line"></div>
+        <p class="global-paragraph">It's been two years and we have recorded great success and achievement</p>
+
+        <div class="image-container">
+          <div class="img-bg-1">
+            <div class="img-template-2">
+              <div class="img-text">
+                <h5 class="alpha-text">2023</h5>
+                <p>Latest Processor</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="img-bg-2">
+            <div class="img-template-2">
+              <div class="img-text">
```

Comment:
- [ ] Good job trying to blur the image as requested by the previous reviewer, however, I think it would be better to reduce the a little bit more the opacity of it to match the Behance design.
- Your design:
![Yblur](https://user-images.githubusercontent.com/97834160/210789687-25d5cf16-702e-46c2-909a-78b9a0f34233.png)

- Behance design:
 
![Bblur](https://user-images.githubusercontent.com/97834160/210789655-faa8cf4b-369c-49db-aab6-3b4b9d158426.png)